### ISSUE TITLE
Firefox developer edition. ": import declarations may only appear at top level of a module

### ISSUE THREAD

**Issue by nuzulfikrie** [ckeditor/ckeditor5-build-classic#87]

**Comment by nuzulfikrie** [comment]
Hi , I cloned the git repository , and when i try load the javascript file via line 
```html 
	<script src="asset/js/ckeditor5-build-classic/src/ckeditor.js"></script> 
```
Firefox complained 
_"import declarations may only appear at top level of a module"_

But when i load from build 
```html 
<script src="asset/js/ckeditor5-build-classic/build/ckeditor.js"></script>
```

The editor able to be rendered on the webpage with all configured plugins loaded. 

I followed the method described in https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/installing-plugins.html.

Did i missed anything out?
